src/test/java/easytests/core/entities/UserEntityTest.java

```diff
@@ -19,7 +19,7 @@ public void testCommon() throws Exception {
 
     @Test
     public void testMap() throws Exception {
-        final UserModelInterface userModel = this.usersSupport.getModelAdditionalMock(0);
+        final UserModelInterface userModel = this.usersSupport.getModelFixtureMock(0);
```

Comment:
@malinink 
Why it was wrong?

Reply:
@Ielay This is not wrong, but previously we don't check id (it is `null` in additionalMock 0 ).
So, it will be better to check with identified model, to be sure that map works fine with id too.

Reply:
@malinink
Got it.